app/javascript/components/admin/AdminTable.js

```diff
@@ -600,6 +600,7 @@ class AdminTable extends React.Component {
   render() {
     return (
       <div className="mx-2">
+        <h1>Admin Dashboard</h1>
```

Comment:
I added the `sr-only` class here - would you rather it removed?

Reply:
I don't know how screen readers work so maybe that's why - but why do we need extra hidden elements rather than the existing elements that already have this kind of description? Is it because screen readers look for `<h1>` elements in particular?

Reply:
I don't know a ton about screen readers but I'm pretty sure they read the page by element hierarchy and not in the order that it is written, which is why `<h1>` tags are important